FHEM/14_SD_WS07.pm

```diff
@@ -37,13 +37,15 @@ SD_WS07_Initialize($)
 {
   my ($hash) = @_;
 
-  $hash->{Match}     = "^P7#[A-Fa-f0-9]{6}F[A-Fa-f0-9]{2}";    ## pos 7 ist aktuell immer 0xF
+  $hash->{Match}     = "^P7#[A-Fa-f0-9]{6}F[A-Fa-f0-9]{2}(#R[A-F0-9][A-F0-9]){0,1}\$";    ## pos 7 ist aktuell immer 0xF
```

Comment:
Wozu muss der RSSI Wert in eine capturing group ? Ich würde bei Hex Werten immer groß und klein Buchstaben akzeptieren.

Gibt es überhaupt einen Grund hier den Match auf den RSSI Wert zu machen?

Reply:
Eigentlich nicht, zumal ja nicht immer ein RSSI dabei ist.